CONTRIBUTORS.md

```diff
@@ -25,3 +25,4 @@
 ### Individual Contributors
 
 Highlight.js has also been greatly improved over the years thanks to the help of [many other contributors](https://github.com/highlightjs/highlight.js/graphs/contributors).  A big thank you to everyone who has helped make this project what it is today!
+- Md Saad Akhtar <akhtarmdsaad@gmail.com>
```

Comment:
Please instead add yourself to the changelog around line ~60... the only contributors mentioned here are former maintainers, everyone else can be found in the GitHub contributor graph.  We no longer maintained a textual list since Github does this for us.

```suggestion
```